cfme/intelligence/chargeback/rates.py

```diff
@@ -57,6 +57,9 @@ class fields(ParametrizedView):  # noqa
         PARAMETERS = ('name',)
         ROOT = ParametrizedLocator('.//tr[./td[contains(normalize-space(.), {name|quote})]]')
 
+        def __locator__(self):
+            return self.browser.element(self.ROOT, parent=self.parent)
```

Comment:
Indeed there is no other option, @izapolsk and I found this as the best work around for now. I can set a comment for that if needed.